ouroboros-consensus-byron/ouroboros-consensus-byron.cabal

```diff
@@ -90,7 +90,7 @@ executable db-converter
     , optparse-generic
     , ouroboros-consensus            ^>=0.4
     , ouroboros-consensus-byron
-    , ouroboros-consensus-diffusion  ^>=0.4
+    , ouroboros-consensus-diffusion  (^>=0.4 || ^>=0.5)
```

Comment:
I was going to suggest
```haskell
    , ouroboros-consensus-diffusion  ^>=0.4 || ^>=0.5
```
but `cabal-fmt` turns that into
```haskell
    , ouroboros-consensus-diffusion  >=0.4  && <0.5  || ^>=0.5
```
right?

Reply:
why not just:
```
, ouroboros-consensus-diffusion  >=0.4 && <0.6
```